docs/zh_cn/04-developer-guide/do_regression_test.md.md

```diff
@@ -6,41 +6,60 @@
 <!-- TOC -->
 
 - [如何进行回归测试](#如何进行回归测试)
-    - [1. 用法](#1-用法)
+    - [1. 环境搭建](#1-环境搭建)
+        - [MMDeploy的安装及配置](#MMDeploy的安装及配置)
+        - [Python环境依赖](#Python环境依赖)
+    - [2. 用法](#2-用法)
         - [参数解析](#参数解析)
         - [示例](#示例)
-    - [2. 回归测试配置文件](#2-回归测试配置文件)
+    - [3. 回归测试配置文件](#3-回归测试配置文件)
         - [示例及参数解析](#示例及参数解析)
-    - [3. 生成的报告](#3-生成的报告)
+    - [4. 生成的报告](#4-生成的报告)
         - [模板](#模板)
         - [示例](#示例)
-    - [4. 支持的后端](#4-支持的后端)
-    - [5. 支持的Codebase及其Metric](#5-支持的Codebase及其Metric)
-    - [6. 注意事项](#7-注意事项)
-    - [7. 常见问题](#8-常见问题)
+    - [5. 支持的后端](#4-支持的后端)
+    - [6. 支持的Codebase及其Metric](#5-支持的Codebase及其Metric)
+    - [7. 注意事项](#7-注意事项)
+    - [8. 常见问题](#8-常见问题)
 
 <!-- TOC -->
 
-## 1. 用法
+
+## 1. 环境搭建
+
+### MMDeploy的安装及配置
+本章节的内容，需要提前根据[build 文档](../build.md)将 MMDeploy 安装配置好之后，才能进行。
+
+### Python环境依赖
+需要安装 test 的环境
+```shell
+pip install -r requirements/tests.txt
+```
+如果在使用过程是 numpy 报错，则更新一下 numpy
+```shell
+pip install -U numpy
+```
+
+## 2. 用法
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml "${DEPLOY_YML_PATH}" \
+    --codebase "${CODEBASE_NAME}" \
     --backends "${BACKEND}" \
     --work-dir "${WORK_DIR}" \
     --device "${DEVICE}" \
     --log-level INFO \
-    [--performance]
+    [--performance 或 -p]
 ```
 
 ### 参数解析
 
-- `--deploy-yml` : 需要测试的 codebase，eg.`configs/mmdet/mmdet_regression_test.yaml`，如果设置为 `all` 即全部测试。
-- `--backends` : 筛选测试的后端, 默认 `all`: 测全部`backend`, 也可传入若干个后端，例如 `onnxruntime tesnsorrt`。
+- `--codebase` : 需要测试的 codebase，eg.`mmdet`, 测试多个 `mmcls mmdet ...`
+- `--backends` : 筛选测试的后端, 默认 `all`: 测全部`backend`, 也可传入若干个后端，例如 `onnxruntime tesnsorrt`。如果需要一同进行 SDK 的测试，需要在 `${codebase}_regression_test.yaml` 里面的 `sdk_config` 进行配置。
```

Comment:
filename has changed for `${codebase}_regression_test.yaml`

Reply:
Done